.autover/changes/e537d29a-c69e-46f0-a857-68c6a75b23d6.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Aspire.Hosting.AWS",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Use Aspire 9\u0027s WaitFor mechanism for waiting CloudFormation resource to be running"
```

Comment:
nit: you might want to remove the unicode char since that's how it'll look in the changelog

Reply:
Removed the Unicode character